apollo-biz/src/main/java/com/ctrip/framework/apollo/biz/service/BizDBPropertySource.java

```diff
@@ -57,7 +57,7 @@ String getCurrentDataCenter() {
   }
 
   @Override
-  protected void refresh() {
+  public void refresh() {
```

Comment:
I think we could make the `com.ctrip.framework.apollo.common.config.RefreshablePropertySource#refresh` method public.

Reply:
@nobodyiam fixed